src/osx.md

```diff
@@ -1,12 +1,17 @@
 # Mac OS X
 
-For now, the easiest way to get going is to install Linux on a virtual
-machine, and follow the [Linux instructions](./linux.html).
+The tools you need are similar to the tools listed in the [Linux instructions](./linux.html), but we will need to build a cross compiler in order to get things working. This is sort of complicated and boring, but we've done some of that boring work for you so you can get up and running quickly.
 
-Otherwise, to install the other tools on Mac OS X, make sure to have [homebrew](http://brew.sh/) installed.
+Make sure you have [homebrew](http://brew.sh/) installed because you are going to need it to install some of the tools. You are probably also going to need [Xcode](https://developer.apple.com/xcode/download/) if you don't already have it.
 
-Then you can run the following command:
+Download [this script](https://gist.githubusercontent.com/emkay/a1214c753e8c975d95b4/raw/cc73290537898799fe1168edfbc5a41351184670/build-grub-osx.sh) and *read* through it. There are a couple assumptions that it makes that you might want to change. When possible we try and use `brew` to install things. The compiled special versions of the tools are installed in `~/opt`. This is so we don't clobber any version of them that you may have already installed. The source code for these tools are downloaded in `~/src`.
```

Comment:
> There are a couple assumptions that it makes that you might want to change.

such as? It'd be awesome to get an idea so folk know what to look out for, best not to assume too much knowledge here :)